programs/bpf_loader/src/syscalls.rs

```diff
@@ -1575,8 +1562,8 @@ fn call<'a>(
             signers_seeds_len,
             memory_mapping,
         )?;
-        let keyed_account_refs = syscall
-            .get_callers_keyed_accounts()
+        let keyed_account_refs = invoke_context
```

Comment:
I'm not following how we can iter here but not use `next_keyed_account` in native program's `process_instruction`, can you detail what the conflict is?

Reply:
Because there are no calls which pass `invoke_context` deeper.
That would implicitly incur a reference to `keyed_accounts` which would collide with the iterator.